### DISCUSSION TITLE
更新1.8后运行还是自动生成.ini，没有使用同路径的yml

### DISCUSSION THREAD

**Discussion by Lue0353** [Yuukiy/JavSP#400] [General]
### 问题详情

更新1.8后运行还是自动生成.ini，没有使用同路径的yml，下载封面也失败

### 运行方式

我运行的是打包后的exe程序

### 代理

是

### 提交须知

- [X] 我确认使用的是最新版本并且阅读过Wiki帮助文档
- [X] 我确认已经搜索过Issue区，没有与我遇到的情况相同的Issue。

**Comment by glyh** [comment]
1.8比Master要旧，很多新功能并没有release（其中包括配置迁移到yml）。如果你想使用这些功能请直接使用master branch下的代码。

That said，代码重构还没有完成，可能会有breaking changes。
